charts/tenantfilesystem/README.md

```diff
@@ -0,0 +1,76 @@
+# tenantfilesystem
+
+![Version: 0.3.0](https://img.shields.io/badge/Version-0.3.0-informational?style=flat-square)
+
+Helm Chart for installing a custom tenantfilesystem xrd
+
+## Installation
+Installs custom wadtfy tenantfilesystem crossplane XRD on an existing Kubernetes cluster for a specific namespace.
+
+**Preconditions**:
+* [Crossplane](https://crossplane.io) need to be installed on Kubernetes.
+* The default [AWS Crossplane Provider](https://github.com/crossplane-contrib/provider-aws) is used for provisioning AWS Resources.
+* At least three AWS availability zones in the VPC are required to establish an EFS connectivity.
+
+### Add Helm repository
+
+```shell
+helm repo add wadtfy-custom-component https://dvpe-cloud.github.io/wadtfy-custom-components
+helm repo update
+```
+
+## Install wadtfy tenantfilesystem chart
+
+```sh
+helm install tenantfilesystem . -f values.yaml
+```
+
+## Use TenantFileSystem to create Filesystems
+
+Create a `tenantfilesystem.yaml` and set the Tenant Account related values
+
+Sample:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: tenantfilesystem
+metadata:
+  name: name
+  namespace: namespace
+spec:
+  compositionRef:
+    name: tenant-filesystem
+  # Tenant AWS Account ID
+  accountID: ""
```

Comment:
accountID, product, region and providerConfig should not be used in this component anymore. Instead use a tenant's pre-provisioned EnvironmentConfig to read those values from

Reply:
Hi @mithie ,

For this change, i need way to handover the EnvironmentConfigs for the Tenant inside the Claim it self:
Because the EnvironmentConfig is set in Composition:

```yaml
apiVersion: apiextensions.crossplane.io/v1
kind: Composition
:
spec:
  environment:
    environmentConfigs:
`

Reply:
please look at https://github.com/DVPE-cloud/wadtfy-custom-components/tree/master/charts/irsa where different EnvironmentConfigs are used based upon label selectors
The EnvironmentConfig will be provisioned together with a Tenant resource, then you could just use it ootb. Prerequisite for using the EFS custom component also will be to have a tenant resource provisioned before.